doc/release-notes/release-6.sgml

```diff
@@ -130,10 +141,16 @@ This section gives an account of those changes in three categories:
 	<p>This feature has been of limited use since AsyncCalls feature
 	   took over much of the CBDATA functionality.
 
+	<tag>--enable-gnuregex</tag>
+	<p>The GnuRegex library has been removed. C++17 standard library is used instead.
+
 	<tag>--enable-kill-parent-hack</tag>
 	<p>This feature has been deprecated for years. Other features such as
 	   <em>--foreground</em> command line argument should be used instead.
 
+	<tag>--enable-leakfinder</tag>
+	<p>Replaced by Valgrind leak detection.
```

Comment:
Valgrind was already supported, so we did not really _replace_ one with the other.

```suggestion
	<p>Removed. Using Valgrind for leak detection is still supported.
```